bm-persona/Drawer/DrawerTabBar/TabBarControl.swift

```diff
@@ -87,14 +87,27 @@ class TabBarControl: UISegmentedControl {
         segments = segments.filter { $0 != indicator }
         layoutIfNeeded()
         widths = []
+        setLabelsAdjustFont(view: self)
         for i in 0..<min(segments.count, numberOfSegments) {
-            widths.append(segments[i].frame.size.width)
+            widths.append(min(segments[i].frame.size.width, maxWidth / CGFloat(segments.count)))
```

Comment:
Wondering if you'd need to do the `min(..., maxWidth / CGFloat(segments.count))` if you moved the `layoutIfNeeded()` call to after you call `setLabelsAdjustFont(...)`.